$MirOS: ports/games/frozen-bubble/patches/patch-doc_frozen-bubble_6,v 1.1 2005/06/24 17:50:05 tg Exp $
--- doc/frozen-bubble.6.orig	Fri Feb 14 21:45:18 2003
+++ doc/frozen-bubble.6	Mon Nov  6 19:06:08 2006
@@ -12,7 +12,6 @@
 .\" along with this program; if not, write to the Free Software
 .\" Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
 .\"
-
 .TH FROZEN-BUBBLE 6 "February 2, 2003" "FB" "The Frozen-Bubble game"
 
 .SH NAME
